docs/tutorials/my-first-custom-action.rst

```diff
@@ -0,0 +1,256 @@
+
+Use this useful debugging commands to make sure your action ( ``report_scheduling_failure``) is loaded:
+
+.. code-block:: bash
+
+    robusta logs # get robusta logs, see errors
+    robusta playbooks list-dirs  # get see if you custom action package was loaded
+
+Let’s push the new action to Robusta, and then test it by triggering the action manually immediately.
+
+.. code-block:: bash
+
+    robusta playbooks push <PATH_TO_LOCAL_PLAYBOOK_FOLDER>
+    robusta playbooks trigger report_scheduling_failure name=robusta-runner-8cd69f7cb-g5bkb namespace=default seconds=5
+
+Check our slack channel, and:
+
+.. image:: /images/example_report_scheduling_failure.png
+
+Connection the trigger to the action - a Playbook is born!
+-------------------------------------
+
+We need to add a custom playbook that this action it in the generated_values.yaml.
+
+.. code-block:: yaml
+
+    globalConfig:
+      signing_key: XXXX
+      account_id: XXXX
+    sinksConfig:
+    - slack_sink:
+        name: main_slack_sink
+        slack_channel: '#my-slack-channel'
+        api_key: XXXXX
+    - robusta_sink:
+        name: robusta_ui_sink
+        token: XXXXXX  # generated with `robusta gen-config`
+    clusterName: my-cluster
+    enablePrometheusStack: true
+    # Custom Playbooks from here
+    customPlaybooks:
+    - triggers:
+      - on_event_create: {}
+      actions:
+      - report_scheduling_failure: {}
+    # Enable loading playbooks to a persistent volume
+    playbooksPersistentVolume: true
+
+.. note::
+    If you haven't already, make sure to clean up the pod from the last section by running ``kubectl delete pod nginx``
+
+Time to update Robusta’s config with the new generated_config.yaml:
+
+.. code-block:: bash
+
+    helm upgrade robusta robusta/robusta --values=generated_values.yaml
+    robusta playbooks list # see all the playbooks. Run it after a few minutes
+
+After a minute or two Robusta will be ready.
+
+Let’s push the new action to Robusta:
+
+.. code-block:: bash
+
+    robusta playbooks push <PATH_TO_PLAYBOOK_FOLDER>
+
+After a minute or two Robusta will be ready.
+
+Great!
+
+Run the scenario from the first section again (creating a bad bad configuration), and you should see this in your slack:
+
+Check our slack channel, and:
+
+.. image:: /images/example_report_scheduling_failure.png
+
+Cleaning up
+----
+
+.. code-block:: bash
+
+    kubectl delete pod nginx # delete the pod
+    robusta playbooks delete <PLAYBOOK_FOLDER> # remove the playbook we just added from Robusta
+
+    # Remove "customPlaybooks" and "playbooksPersistentVolume" from you config, and then run helm upgrade
+    helm upgrade robusta robusta/robusta --values=generated_values.yaml
+
+
+Summary
+-------------------------------------
+
+We learned how to solve a real problem (pod not scheduling) only once and have Robusta automate it in the future for all our happy co-workers (and future us) to enjoy.
```

Comment:
Should we add something here like:

This example of an unschedulable pod is actually covered by Robusta out of the box (if you enable the builtin Prometheus stack) but you can see how easy it is to track any error you like and send it to a notifications system with extra data.